data/hooks/conf_regen/15-nginx

```diff
@@ -38,6 +38,15 @@ do_pre_regen() {
   for domain in $domain_list; do
     domain_conf_dir="${nginx_conf_dir}/${domain}.d"
     mkdir -p "$domain_conf_dir"
+    domain_ca_type=$(yunohost domain cert-status ${domain} | grep CA_type | tr -d " ")
+    if [[ "$domain_ca_type" = "CA_type:Let'sEncrypt" ]]; then
+      if [ ! -f "/etc/yunohost/certs/${domain}/chain.pem" ];then
+        wget -O ${pending_dir}/etc/yunohost/certs/${domain}/chain.pem "https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem"
+      fi
+      sed "s/#ssl_stapling/ssl_stapling/g" "${nginx_conf_dir}/${domain}.conf"
+      sed "s/#ssl_trusted_certificate/ssl_trusted_certificate/g" "${nginx_conf_dir}/${domain}.conf"
```

Comment:
As discussed on the chat, I'd really like if we could rely on `ynh_render_template` here instead of sed's :/

Reply:
Well, it's just two sed... not really important (as we discussed in the chat). I don't know how ynh_render_template works... but why not.